lib/badfruit/Movies/movie.rb

```diff
@@ -26,6 +26,11 @@ def full_cast
       return @badfruit.parse_actors_array(JSON.parse(@badfruit.get_movie_info(@id, "cast")))
     end
 
+    # Returns director, since RottenTomatoes API is broken (abridged_director doesn't show up in search_by_name results for movies.)
+    def director
+      return Director.new(JSON.parse(@badfruit.get_movie_info(@id, "details"))["abridged_directors"][0])
```

Comment:
So I feel like this data should be cached on the object after it has been retrieved. Also, there is a possibility to multiple directors on a single move. Take a look at the movie with the id `460859831` (Grindhouse) so this property should probably be called `directors` and return an array of all the directors of a movie.

I'm also trying to reach out to Rotten Tomatoes since I'd like for them to fix this in the API.